db/migrate/20210404213440_add_notes_to_users.rb

```diff
@@ -0,0 +1,12 @@
+class AddNotesToUsers < ActiveRecord::Migration[6.1]
+    def up
+      add_column :users, :notes, :text
+      User.all.each do |user|
+        user.update(notes: user.notes || '')
+      end
```

Comment:
Solution to slow migration is to use `update_all` (and probably have a `before_save` callback in the user model to set nil notes to `''`) or to allow nil for user notes.

Reply:
The performance DB has ~80k users so iterating over each one and updating is not viable

Reply:
100% agree with @jkufro , but I think we can also just set a default value by using:
```
add_column :users, :notes, :text, default: ''
```

Reply:
that way, we can also just define the entire migration by:
```
def change
  add_column :users, :notes, :text, default: ''
end
```

Reply:
Unfortunately we can't set a default value for a text field but the update_all seems to have fixed the migration error.

Reply:
alternative to not being able to set a default at the migration level is to have a `before_save` callback that ensures that the `notes` field is not nil

Reply:
Ah got it, because it's a `text` field and not a `string`. Does it even need to have a default value though? I don't think our other `notes` fields such as `exposure_notes` on patients have/need default values?

Reply:
Removed the update in  https://github.com/SaraAlert/SaraAlert/pull/805/commits/6692b2c0557b7786044f99801d4bce25b73e6047